erpnext/accounts/utils.py

```diff
@@ -283,18 +282,22 @@ def get_balance_on(
 		cond.append("""gle.company = %s """ % (frappe.db.escape(company, percent=False)))
 
 	if account or (party_type and party) or account_type:
-
+		precision = frappe.db.escape(get_currency_precision())
```

Comment:
```suggestion
		precision = get_currency_precision()
```

Now that we are passing parameters to the database, so we don't need to escape them anymore. In fact, this produces wrong behavior (always rounding to 0 decimal places, because a string is not a valid number of decimals).